src/main/java/graphql/execution/ExecutionContextBuilder.java

```diff
@@ -78,30 +71,25 @@ public ExecutionContextBuilder root(Object root) {
         return this;
     }
 
-    public ExecutionContextBuilder document(Document document) {
-        this.document = document;
+    public ExecutionContextBuilder variables(Map<String, Object> variables) {
+        this.variables = variables;
         return this;
     }
 
-    public ExecutionContextBuilder operationName(String operationName) {
-        this.operationName = operationName;
+    public ExecutionContextBuilder fragmentsByName(Map<String, FragmentDefinition> fragmentsByName) {
+        this.fragmentsByName = fragmentsByName;
         return this;
     }
 
-    public ExecutionContextBuilder variables(Map<String, Object> variables) {
-        this.variables = variables;
+    public ExecutionContextBuilder operationDefinition(OperationDefinition operationDefinition) {
+        this.operationDefinition = operationDefinition;
         return this;
     }
 
     public ExecutionContext build() {
         // preconditions
         assertNotNull(executionId, "You must provide a query identifier");
 
```

Comment:
This has been moved up and out into `Execution`.  See above

Both are @Internal code so its safe